impl/src/main/java/com/okta/sdk/impl/resource/AbstractResource.java

```diff
@@ -102,7 +102,7 @@ public void setInternalProperties(Map<String, Object> properties) {
                 } else {
                     this.properties = properties;
                 }
-                setResourceHref(getSelfHref(properties));
+                setResourceHref(hrefResolver.resolveHref(properties, getClass()));
```

Comment:
`resolveHref` can return `null` right? Do we need to handle the case when its `null`

Reply:
Nope `hrefResolver` is set in the constructor and is final

Reply:
sorry, i read that comment wrong.
Yes, it can return `null`, but that fine. For example in the case of a password reset body there would be no HREF (there is no way to request that resource again)